### ISSUE TITLE
Discord Bitbucket bot not updating ?

### ISSUE THREAD

**Issue by rafael-plugge** [Commit451/skyhook#16]
Been using Skyhook to receive bitbucket repo push's and all issue updates, could you look into this please.

```
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Error</title>
</head>
<body>
<pre>ReferenceError: user is not defined<br> &nbsp; &nbsp;at Object.parse (/app/providers/bitbucket.js:146:22)<br> &nbsp; &nbsp;at /app/server.js:46:23<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)<br> &nbsp; &nbsp;at next (/app/node_modules/express/lib/router/route.js:137:13)<br> &nbsp; &nbsp;at Route.dispatch (/app/node_modules/express/lib/router/route.js:112:3)<br> &nbsp; &nbsp;at Layer.handle [as handle_request] (/app/node_modules/express/lib/router/layer.js:95:5)<br> &nbsp; &nbsp;at /app/node_modules/express/lib/router/index.js:281:22<br> &nbsp; &nbsp;at param (/app/node_modules/express/lib/router/index.js:354:14)<br> &nbsp; &nbsp;at param (/app/node_modules/express/lib/router/index.js:365:14)<br> &nbsp; &nbsp;at param (/app/node_modules/express/lib/router/index.js:365:14)</pre>
</body>
</html>
```

**Comment by rafael-plugge** [comment]
If anymore info is need please ask (I'm pretty new to this)

**Comment by Jawnnypoo** [comment]
Fixed with PR #17
